packages/server/.env.development

```diff
@@ -25,4 +25,10 @@ Z4UMR7EOcpfdUE9Hf3m/hs+FUR45uBJeDK1HSFHD8bHKD6kv8FPGfJTotc+2xjJw
 oYi+1hqp1fIekaxsyQIDAQAB
 -----END PUBLIC KEY-----'
 
-APPLY_PASSWORD=chungus
\ No newline at end of file
+APPLY_PASSWORD=chungus
+KHOURY_ADMIN_OAUTH_API_URL=https://admin-alpha.khoury.northeastern.edu/api/oauth
+KHOURY_ADMIN_OAUTH_URL=https://admin-alpha.khoury.northeastern.edu/oauth
+OAUTH_CLIENT_ID=10aac8aac0f1f711756f
```

Comment:
i hope these arent very important (i think the PR comment said something about these being dummy values but I want to make sure lol)